addons/survey/static/src/js/fields_section_one2many.js

```diff
@@ -39,6 +39,9 @@ var SectionListRenderer = ListRenderer.extend({
 
         if (isSection) {
             if (node.attrs.widget === "handle" || node.attrs.name === "random_questions_count") {
+                if (node.attrs.name === "random_questions_count"){
+                    $cell.attr('colspan', 1 + this.columns.filter(elem => elem.tag === "button_group").length);
+                }
```

Comment:
It looks like the code for v15.2 works well for v14.0 too
Why don't you use it ? Does your code has something extra ?

Reply:
@pko-odoo 
You are right, I think at first I thought it hadn't worked but when I retested, it worked so here's the propagation of the fix from 15.2